packages/remix-react/transition.ts

```diff
@@ -590,7 +590,7 @@ export function createTransitionManager(init: TransitionManagerInit) {
         invariant(matches, "No matches found");
         if (fetchControllers.has(key)) abortFetcher(key);
 
-        let match = getFetcherRequestMatch(
+        let match = getRequestMatch(
```

Comment:
We now use the same util for action `submissions` and fetchers - so we only do this `?index`/`pathless` logic once